terraform/modules/gro_ingestion_service/lambdas/src/functions/enrichEvent.function.ts

```diff
@@ -1,14 +1,13 @@
-import { Handler } from "aws-lambda"
 import { DocumentClient } from "aws-sdk/clients/dynamodb"
 import { mapToEventRecord } from "../models/EventRecord"
 import { EnrichEventRequest } from "../models/EnrichEventRequest"
 import { EnrichEventResponse } from "../models/EnrichEventResponse"
 
-const tableName = process.env.TABLE_NAME ?? ""
+const tableName: string = process.env.TABLE_NAME ?? ""
```

Comment:
I don't think we should be enforcing explicit typing, this is inferred easily for a lot of these